dev.c

```diff
@@ -597,14 +597,16 @@ struct fuse_req *request_find(struct fuse_conn *fc, u64 unique)
 	return req;
 }
 
-struct fuse_req* request_find_in_ctx(unsigned ctx, u64 unique)
+/*
+static struct fuse_req* request_find_in_ctx(unsigned ctx, u64 unique)
```

Comment:
can we leave it uncommented please, it is used in master and not in release branch yet.

Reply:
@sulakshm I am sorry but the reason I commented this out was because of this error.   If we leave it in then we will need to modify the makefile to not error on this "-Wno-unused-function".  Unless I can just add a call to it?
```
make  -C /usr/src/linux-headers-6.8.0-7-generic  M=/source modules
make[1]: Entering directory '/usr/src/linux-headers-6.8.0-7-generic'
warning: the compiler differs from the one used to build the kernel
  The kernel was built by: x86_64-linux-gnu-gcc-13 (Ubuntu 13.2.0-13ubuntu1) 13.2.0
  You are using:           gcc-13 (Ubuntu 13.2.0-4ubuntu3) 13.2.0
Kernel version 6.8 supports fastpath.
Kernel version 6.8 supports blkmq driver model.
  CC [M]  /source/pxd.o
  CC [M]  /source/dev.o
/source/dev.c:600:25: error: 'request_find_in_ctx' defined but not used [-Werror=unused-function]
  600 | static struct fuse_req* request_find_in_ctx(unsigned ctx, u64 unique)
      |                         ^~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[3]: *** [scripts/Makefile.build:243: /source/dev.o] Error 1
make[2]: *** [/usr/src/linux-headers-6.8.0-7-generic/Makefile:1926: /source] Error 2
make[1]: *** [Makefile:240: __sub-make] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-6.8.0-7-generic'
make: *** [Makefile:154: all] Error 2
```

Reply:
ok can you leave it commented atleast.